indexd/index/blueprint.py

```diff
@@ -176,6 +177,12 @@ def post_index_record():
     except jsonschema.ValidationError as err:
         raise UserError(err)
 
+    did = flask.request.json.get('did')
+
+    if did is not None:
+        if DID_PATTERN(did) is None:
```

Comment:
this is already validated by [jsonschema](https://github.com/uc-cdis/indexd/blob/4ef1fbcd8a267d920523d7dab98c0a921dbffc87/indexd/index/schema.py#L49) ?

Reply:
Yeah. This is redundant. Need to be removed. Thanks